epoxy-adapter/src/main/java/com/airbnb/epoxy/stickyheader/StickyHeaderLinearLayoutManager.kt

```diff
@@ -0,0 +1,621 @@
+package com.airbnb.epoxy.stickyheader
+
+import android.content.Context
+import android.graphics.PointF
+import android.os.Build
+import android.os.Parcelable
+import android.view.View
+import android.view.ViewGroup
+import android.view.ViewTreeObserver
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.airbnb.epoxy.BaseEpoxyAdapter
+import kotlinx.android.parcel.Parcelize
+
+/**
+ * Adds sticky headers capabilities to your [RecyclerView.Adapter].
+ * The adapter must implement [HasStickyHeader] to indicate which items are
```

Comment:
re `The adapter must implement [HasStickyHeader]` is this necessary to call out since epoxy base classes already implement it? is it confusing to mention this?

Reply:
Makes sense. I'll reword it. Initially the base classes were not implementing it hence it was mentioned.

Reply:
Turns out, the adapter / controller needs to implement the `isStickyHeader` function so I'll keep it comment as it is and just tweak it a bit and add a example usage in the code docs